components/button/button-move.js

```diff
@@ -49,11 +51,41 @@ class ButtonMove extends FocusMixin(RtlMixin(LitElement)) {
 			 * @type {boolean}
 			 */
 			disabled: { type: Boolean, reflect: true },
+			/**
+			 * Disables the down interaction
+			 * @type {boolean}
+			 */
+			disabledDown: { type: Boolean, attribute: 'disabled-down' },
+			/**
+			 * Disables the end interaction
+			 * @type {boolean}
+			 */
+			disabledEnd: { type: Boolean, attribute: 'disabled-end' },
+			/**
+			 * Disables the home interaction
+			 * @type {boolean}
+			 */
+			disabledHome: { type: Boolean, attribute: 'disabled-home' },
+			/**
+			 * Disables the left interaction
+			 * @type {boolean}
+			 */
+			disabledLeft: { type: Boolean, attribute: 'disabled-left' },
+			/**
+			 * Disables the right interaction
+			 * @type {boolean}
+			 */
+			disabledRight: { type: Boolean, attribute: 'disabled-right' },
 			/**
 			 * Tooltip text when disabled
 			 * @type {string}
 			 */
 			disabledTooltip: { type: String, attribute: 'disabled-tooltip' },
+			/**
+			 * Disables the up interaction
+			 * @type {boolean}
+			 */
+			disabledUp: { type: Boolean, attribute: 'disabled-up' },
```

Comment:
These will all need to `reflect` since you're using CSS attribute selectors down below.